controllers/deviceservice_controller.go

```diff
@@ -173,19 +159,19 @@ func (r *DeviceServiceReconciler) reconcileDeleteDeviceService(ctx context.Conte
 	return nil
 }
 
-func (r *DeviceServiceReconciler) reconcileCreateDeviceService(ctx context.Context, ds *devicev1alpha1.DeviceService, log logr.Logger) error {
+func (r *DeviceServiceReconciler) reconcileCreateDeviceService(ctx context.Context, ds *devicev1alpha1.DeviceService) error {
 	// get the actual name of deviceService on the Edge platform from the Label of the device
 	edgeDeviceServiceName := util.GetEdgeDeviceServiceName(ds, EdgeXObjectName)
-	log.V(4).Info("Checking if deviceService already exist on the edge platform", "deviceService", ds.GetName())
+	klog.V(4).Infof("Checking if deviceService already exist on the edge platform: %s", ds.GetName())
 	// Checking if deviceService already exist on the edge platform
 	if edgeDs, err := r.deviceServiceCli.Get(nil, edgeDeviceServiceName, edgeInterface.GetOptions{}); err != nil {
 		if !clis.IsNotFoundErr(err) {
-			log.V(4).Error(err, "fail to visit the edge platform")
+			klog.V(4).ErrorS(err, "fail to visit the edge platform")
 			return nil
 		}
 	} else {
 		// a. If object exists, the status of the device on OpenYurt is updated
-		log.V(4).Info("DeviceService already exists on edge platform")
+		klog.V(4).Info("DeviceService already exists on edge platform")
```

Comment:
Adding deviceservice name into the log message may be better.

Reply:
The deviceservice name has been added.